### ISSUE TITLE
Runtimeenvironment - Will this rund and work with Mono on Linux?

### ISSUE THREAD

**Issue by mabra** [kapetan/dns#100]
A short look into the code says me: yes.
But a confirmation would be better.
Thanks.

**Comment by MiheevN** [comment]
Why Mono?
Do you have a ban on installing dotnet itself?
Everything just works fine in dotnet.

**Comment by mabra** [comment]
No, no ban, but too old environment - which will be replaced later and there are some strong dependencies, like loadung and unloading dll's at runtime into separate AppDomains.
BTW, a question ist not an answer ;-)
